src/v/kafka/client/broker.cc

```diff
@@ -22,9 +23,13 @@ ss::future<shared_broker_t> make_broker(
   model::node_id node_id,
   unresolved_address addr,
   const configuration& config) {
+    auto creds
+      = co_await cluster::maybe_build_reloadable_certificate_credentials(
+        config.broker_tls());
     auto client = ss::make_lw_shared<transport>(
-      rpc::base_transport::configuration{.server_addr = addr});
-    return client->connect()
+      rpc::base_transport::configuration{
+        .server_addr = addr, .credentials = std::move(creds)});
+    co_return co_await client->connect()
```

Comment:
what's the point of this co-await co-return, doesn't seem like anything needs this ?

Reply:
`[[deprecated("Forwarding coroutine returns are deprecated as too dangerous. Use 'co_return co_await ...' until explicit syntax is available.")]]`

Reply:
got it.